examples/benchmark/clue/mrc/run_chid.py

```diff
@@ -526,9 +526,10 @@ def add_tokens_for_around(tokens, pos, num_tokens):
                                loss, args.logging_steps /
                                (time.time() - tic_train)))
                         tic_train = time.time()
-                if global_step >= num_training_steps:
-                    logger.info("best_result: %.2f" % (best_acc * 100))
-                    return
+                if global_step > num_training_steps:
+                    break
+            if global_step > num_training_steps:
+                break
```

Comment:
```suggestion
                if global_step >= num_training_steps:
                    break
            tic_eval = time.time()
            acc = evaluate(model, dev_data_loader)
            logger.info("" )
            ## save ckpt
            if global_step >= num_training_steps:
                break
```

这样是不是合理一些，现在的逻辑是每个 epoch 评估一次，没跑到 1个epoch就不会评估，即使是2.9 epoch 也是评估2次。
还有  `global_step > num_training_steps` 是不是多运行了一个step，还是 ` >= ` ?

Reply:
如果阅读理解任务是每1个epoch eval一次，2.9个epoch最后还需要eval一次吗？